packages/tpetra/core/ext/TpetraExt_MatrixMatrix_def.hpp

```diff
@@ -817,6 +821,18 @@ add (const Scalar& alpha,
                               col_inds_array, global_col_inds_array,
                               typename map_type::local_map_type>
         (localColinds, globalColinds, CcolMap->getLocalMap()));
+    //TODO: use KokkosKernels batched sort on device as soon as it's available
```

Comment:
Why not use the function `sortCrsEntries` from `Tpetra_Import_Util2.hpp`?
It would avoid redundant code and potentially bugs...

Reply:
@lucbv Thanks, it does do exactly what I want. Fixed.